Sources/SwiftInfo/main.swift

```diff
@@ -22,7 +27,15 @@ struct Swiftinfo: ParsableCommand {
     @Flag(name: .customLong("print-sourcekit", withSingleDash: true), help: "print source kit")
     var printSourcekit = false
 
+    @Argument(parsing: .unconditionalRemaining)
+    var arguments: [String] = []
```

Comment:
This will basically ignore unknown commands. so users can pass whatever they want and it will just ignore them. since we will be using `CommandLine.arguments` to pass the actual arguments whatever the users wants us to pass them.

Reply:
Can this be called as `swiftinfo -v -myCustomArg` or does it have to be `swiftinfo -v -- -myCustomArg` ?

Reply:
Yeah it can be called, for example: `swiftinfo -pullRequest --clean-build -I out/dir/ -s`. Swiftinfo only parse `pullRequest` and `s` and the rest will be available through `CommandLine.arguments`

Reply:
This is perfect then! 👍  I will run some manual tests tomorrow and merge if everything is fine. Thank you!

Reply:
No worries! if there is anything missing please let me know and I'll fix it. 
and I will try to see if I would be able to pass in an `error` for the termination